index.js

```diff
@@ -630,6 +630,14 @@ instance.prototype.getRequest = function(url, cb) {
 			}
 		} else if (response.statusCode != 200) {
 			self.show_error( { message: response.statusMessage } );
+		} else if (data[0]!='{') {
```

Comment:
What data type is `data`?  I'd expect this evaluation to use a substring call typically.

Reply:
The `node-rest-client.get` function returns `Buffer`. It gets stringified just before the `JSON.parse` (lines 562,563) in the callback/processing function.

Reply:
It didn't make sense to use `data.toString().charAt(0)` (2 extra function calls) to extract the first character since `data[0]` already does that.

Reply:
This is changed to `data[0] != 123` with comments. I don't know why `=='<'` works and `!='{'` doesn't. The test against '{' is more precise.